### PR TITLE
Avatar Color Tokens Update

### PR DESCRIPTION
**Pull request by laminesm**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

Updated the Avatar's color tokens to Fluent 2 tokens. These changes were on the background, the foreground and the ring colors.

### Verification

The changes were tested on the avatars in the demo app.

`No Ring/No Presence`

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| before_light | after_light |
| before_dark | after_dark |

`With Presence`

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| before_light | after_light |
| before_dark | after_dark |

`With Ring`

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| before_light | after_light |
| before_dark | after_dark |

### Pull request checklist

This PR has considered:
- [x] Light and Dark appearances
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [x] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/1045)

### CODE REVIEW

File: ios/FluentUI/Avatar/AvatarTokens.swift [hunk 1]
```diff
@@ -69,13 +69,13 @@ open class AvatarTokens: ControlTokens {
     open var ringDefaultColor: DynamicColor {
         switch style {
         case .default, .group:
-            return globalTokens.brandColors[.tint10]
+            return aliasTokens.strokeColors[.brandStroke1]
```
Comment (huwilkes) [inline ios/FluentUI/Avatar/AvatarTokens.swift:72]: I think you're missing #1032 , and these should be `aliasTokens.colors[.brandStroke1]` etc

File: ios/FluentUI/Avatar/AvatarTokens.swift [hunk 2]
```diff
@@ -179,15 +179,15 @@ open class AvatarTokens: ControlTokens {
...
-            return globalTokens.brandColors[.primary]
+            return aliasTokens.colors[.brandBackground1]
         case .outlined:
-            return .init(light: globalTokens.neutralColors[.grey94], dark: globalTokens.neutralColors[.grey26])
+            return aliasTokens.colors[.background5]
         case .outlinedPrimary:
-            return .init(light: globalTokens.brandColors[.tint40].light, dark: globalTokens.neutralColors[.grey26])
+            return aliasTokens.colors[.brandBackground4]
         case .overflow:
-            return aliasTokens.backgroundColors[.neutral4]
```
Comment (huwilkes) [inline ios/FluentUI/Avatar/AvatarTokens.swift:190]: nit: for background and foreground, I think we can also put `.overflow` and `.outlined` on the same line, like `ringDefaultColor`.

**Review by huwilkes — APPROVED** [review]

File: ios/FluentUI/Avatar/AvatarTokens.swift (hunk 2 above)
Comment (rdeassis) [reply ios/FluentUI/Avatar/AvatarTokens.swift:190]: Yes! There are other instances of duplicated logic even on the ringDefaultColor (`aliasTokens.colors[.brandStroke1]`)

All of these repeated values can be consolidated in the same case of the switch statement:


```
    open var ringDefaultColor: DynamicColor {
        switch style {
        case .default, .group, .accent,  .outlinedPrimary:
            return aliasTokens.colors[.brandStroke1]
        case .outlined, .overflow:
            return aliasTokens.colors[.stroke1]
        }
    }
```

File: ios/FluentUI/Avatar/AvatarTokens.swift [hunk 3]
```diff
@@ -69,13 +69,13 @@ open class AvatarTokens: ControlTokens {
     open var ringDefaultColor: DynamicColor {
         switch style {
         case .default, .group:
-            return globalTokens.brandColors[.tint10]
+            return aliasTokens.colors[.brandStroke1]
```
Comment (rdeassis) [inline ios/FluentUI/Avatar/AvatarTokens.swift:72]: Have you also validated all the changes in the SwiftUI Demo controller?
We don't expect any issues, but just for the sake of validation (which you can also mark in the validation checklist of the PR).

Comment (laminesm) [reply ios/FluentUI/Avatar/AvatarTokens.swift:72]: Ah I'm sorry I didn't validate the changes on the SwiftUI Demo by the time I made the PR but I just did and everything looks good!

**Review by mischreiber — APPROVED** [review]

**Review by rdeassis — APPROVED** [review]

**Pull request merged**